docs/design-proposals/2021-06-15-add-path-to-tidb-monitor.md

```diff
@@ -0,0 +1,41 @@
+# Pause TiDB Cluster
+
+This document presents a design to add path of tidb monitor in
+operator.
+
+## Motivation
+
+At present, when we want to expose monitoring services (including grafana and prothumes) through Ingress, our design is to use different hosts for different tidb-monitors to distinguish which specific monitoring service is being accessed.This brings some inflexible experiences. If we want to distinguish between different monitoring services through one host but different paths, it is impossible to do it now, but in fact this is also a feasible solution
+
+## Proposal
+
+We can add path to the spec and apply it to the ingress spec application, which can bring us more flexible path parameters [spec code](https://github.com/pingcap/tidb-operator/blob/master/pkg/apis/pingcap/v1alpha1/types.go#L1709)
```

Comment:
I suggest link to a permalink (to a specified git commit, like https://github.com/pingcap/tidb-operator/blob/2c7a0cae64a6ec8ec5e93033203a8087fc22d819/pkg/apis/pingcap/v1alpha1/types.go#L1709) rather than `master`.

if so, ditto for L28.